cmd/gossamer/main.go

```diff
@@ -411,3 +426,37 @@ func buildSpecAction(ctx *cli.Context) error {
 
 	return nil
 }
+
+func pruneState(ctx *cli.Context) error {
+	inputDBPath := ctx.GlobalString(BasePathFlag.Name)
+	if inputDBPath == "" {
+		inputDBPath = dot.GssmrConfig().Global.BasePath
+	}
```

Comment:
the user should be able to specify the chain with `--chain` and then it should use the basepath based on the chain

Reply:
Done